### PR TITLE
Symfony2 hhvm

### PR DESCRIPTION
**Pull request by mcocciaTE**

### CODE REVIEW

File: frameworks/PHP/symfony2/benchmark_config.json [hunk 1]
```diff
@@ -59,6 +59,27 @@
...
+      "query_url": "/db?queries=",
+      "fortune_url": "/fortunes",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Fullstack",
+      "database": "MySQL",
+      "framework": "symfony2",
+      "language": "PHP",
+      "orm": "Full",
+      "platform": "PHP-FPM",
```
Comment (msmith-techempower) [inline frameworks/PHP/symfony2/benchmark_config.json]: This should be `"platform": "hhvm",`

File: frameworks/PHP/symfony2/benchmark_config.json [hunk 2]
```diff
@@ -59,6 +59,27 @@
...
+      "webserver": "nginx",
```
Comment (msmith-techempower) [inline frameworks/PHP/symfony2/benchmark_config.json]: This should be `"webserver": "HPHP",`

File: frameworks/PHP/symfony2/benchmark_config.json [hunk 3]
```diff
@@ -59,6 +59,27 @@
...
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "symfony2",
```
Comment (msmith-techempower) [inline frameworks/PHP/symfony2/benchmark_config.json]: Let's change this to `"display_name": "symfony2-hhvm",`

**Comment by msmith-techempower** [comment]
Looking good - let's see what Travis thinks!

**Comment by mcocciaTE** [comment]
Fixed up config file so I think it should be good to go now

**Pull request merged**